_posts/2018-08-15-nov-hardfork-release.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title: Bitcoin ABC 0.18.0 now available!
+subtitle: Be sure to upgrade before the November hardfork
+---
+
+The goal of Bitcoin ABC is to help create sound money that is usable by everyone in the world. We believe this is a civilization-changing technology which will dramatically increase human freedom and prosperity.
+
+Our role in furthering this goal is to produce high-quality software that serves the needs of the community of users, miners, and merchants. We strive for continuous technical improvement, to produce a reliable product that will provide a solid foundation for Bitcoin Cash. As part of our plan for continuous improvement, we're keeping in line with our [November Upgrade Timeline](https://www.bitcoinabc.org/2018-07-05-nov-hardfork-timeline/).
+
+The Bitcoin ABC release version 0.18.0 is now available.  You can download it at: [https://download.bitcoinabc.org/0.18.0/](https://download.bitcoinabc.org/0.18.0/)
```

Comment:
Do we want to link to the specification of changes?

Reply:
Good idea.  Will do.